R/teal_slice.R

```diff
@@ -586,14 +586,14 @@ justify_json <- function(json) {
   }
 
   json_lines <- strsplit(json, "\n")[[1]]
-  json_lines_split <- strsplit(json_lines, split = ":", fixed = TRUE)
-  name_width <- max(unlist(gregexpr(":", json_lines))) - 1
+  json_lines_split <- regmatches(json_lines, regexpr(":", json_lines), invert = TRUE)
```

Comment:
```suggestion
  json_lines_split <- strsplit(json_lines, split = "^([^,]+),", fixed = TRUE)
```

Reply:
@gogonzo I don't get what is your change doing here? you do not have `:` in the split regex
this is result of your change
`````{R}
> strsplit('"value": "fields with dates 02:04:03"', split = "^([^,]+),", fixed = TRUE)
[[1]]
[1] "\"value\": \"fields with dates 02:04:03\""
````
We actually wanna split by the first appearance of `:`

```{R}
regmatches('"value": "fields with dates 02:04:03"', regexpr(":", '"value": "fields with dates 02:04:03"'), invert = TRUE)
[[1]]
[1] "\"value\""                       " \"fields with dates 02:04:03\""
```

Reply:
Sorry, replace `,` with `:`